### DISCUSSION TITLE
Parameter identification

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2172] [Ideas]
Hello pybamm developers, can pybamm identify battery parameters? For example, I have a set of battery aging data, can pybamm identify battery aging parameters?

**Comment by valentinsulzer** [comment]
The basic building block software for this is available https://youtu.be/oUhOL8ebOFw
But there are many open questions on e.g. identifiability of these parameters, it is definitely not at the "just run this example" stage yet
